main.go

```diff
@@ -39,7 +39,9 @@ func main() {
 	flag.Parse()
 
 	if os.Getenv("TERM_PROGRAM") != "iTerm.app" {
-		fatal("iTerm2 required")
+		if !osc.IsSixelSupported() {
+			fatal("iTerm2 or DRCS Sixel graphics required")
+		}
```

Comment:
Would be better merge those two condition into one:

`if os.Getenv("TERM_PROGRAM") != "iTerm.app" && !osc.IsSixelSupported()`

Reply:
I would also suggest to move those condition into an `osc.HasGraphicsSupport` or something.